cmd/Kconfig

```diff
@@ -1346,6 +1346,13 @@ config CMD_CLONE
 	  initial flashing by external block device without network
 	  or usb support.
 
+config CMD_OPTEE_LOAD
+	bool "Load OP-TEE binary from U-boot"
+	depends on OPTEE_LOAD_IMAGE
```

Comment:
depends on OPTEE_LOAD_IMAGE && CMDLINE ?

Reply:
Also maybe it's a good idea to add this the the defconfig? configs/rpi_arm64_defconfi
I'm not sure because it's shared for at leash rpi4 and rpi5. But it's a good thing to consider/

Reply:
Also I would make this `default y if OPTEE_LOAD_IMAGE` - I see no reason to set them separately